src/audio/play.rs

```diff
@@ -0,0 +1,27 @@
+//! Provides functions used to play audio.
+
+use rodio::*;
+
+use std::io::Cursor;
+
+use super::source::Source;
+use super::output::Output;
+
+/// Play a sound once.
+///
+/// This will return an Error if the loaded audio file in source could not be decoded.
+pub fn once(source: &Source, endpoint: &Output) -> Result<(), ()> {
+    match play_once(&endpoint.endpoint, Cursor::new(source.clone())) {
+        Ok(sink) => {
+            sink.detach();
+            Ok(())
+        }
+
+        /// There is one and only one error that can be returned, which is unrecognized format
+        /// See documentation for DecoderError here:
+        /// https://docs.rs/rodio/0.5.1/rodio/decoder/enum.DecoderError.html
+        Err(_) => {
+            Err(())
```

Comment:
Not `()` :smile: Just think about the case where a user unwraps it or returns `Box<Error>` or similar cases. A unit type `DecoderError` is much more descriptive here.